src/Widgets/HeaderBar.vala

```diff
@@ -19,12 +19,9 @@
 * Authored by: Alessandro "Alecaddd" Castellani <castellani.ale@gmail.com>
 */
 
-public class Akira.Widgets.HeaderBar : Gtk.Box {
+public class Akira.Widgets.HeaderBar : Gtk.HeaderBar {
     private const string TOOLS_DIR = "/com/github/alecaddd/akira/tools/";
 
-    private Gtk.HeaderBar headerbar;
```

Comment:
These items were never actually used outside of the construct method, so there's no need to make them private to the class

Reply:
Oh really?
So, if an item is not used in any method of the class but only in the construct, it doesn't need to be declared before?
Where can I find this explanation in the documentation?

Reply:
Naw you only need to make it private to the class if you need to access it from another method in that class. Otherwise you can just use the `var` keyword. Not sure if that’s specifically documented somewhere or not